src/main/java/me/earth/earthhack/impl/modules/client/hud/HUD.java

```diff
@@ -240,6 +246,25 @@ protected void renderModules() {
             final String dir = RotationUtil.getDirection4D(false);
             renderText(dir, 2, height - 3 - RENDERER.getStringHeightI() * 2 - animationY);
         }
+
+        if (totems.getValue()) {
+            RenderItem itemRender = mc.getRenderItem();
+            int width = resolution.getScaledWidth();
+            int height = resolution.getScaledHeight();
+            int totems = InventoryUtil.getCount(Items.TOTEM_OF_UNDYING);
+
+            if (totems > 0) {
+                int x = width / 2 - 7;
+                int y = height - 40 - (mc.playerController.gameIsSurvivalOrAdventure() ? 15 : 0);
```

Comment:
Can you instead take a look at how the y coordinate is handled for the Armor Hud [here](https://github.com/Ozynt/3arthh4ck/blob/907eb2700d46c3536b687c697a51aea107434cb4/src/main/java/me/earth/earthhack/impl/modules/client/hud/HUD.java#L318)? Right now bubbles and horse hearts are not getting taken into account. Could you extract the lines 318-341 into a getArmorY method and use it here too? So that both in survival and creative the totem and Armor huds are aligned.

Reply:
Last small thing, everything else LGTM!